include/deal.II/base/symmetric_tensor.h

```diff
@@ -3521,7 +3521,216 @@ outer_product(const SymmetricTensor<2, dim, Number> &t1,
   return tmp;
 }
 
+/**
+ * Perform a spectrum decomposition of a 2nd-order symmetric tensor \a
```

Comment:
doxygen allows both `\a` and `@a` for markup (and the same for all other doxygen commands). We use `@` in our documentation, and leave `\` for latex markup. Would you mind switching to using `@`?

Reply:
It is switched to @a.